core/web/resolver/mutation.go

```diff
@@ -171,6 +172,13 @@ func (r *Resolver) CreateFeedsManagerChainConfig(ctx context.Context, args struc
 		return nil, err
 	}
 
+	if args.Input.AdminAddr != "" {
```

Comment:
This validation should be on the service methods since it is considered domain logic